packages/mwp-app-render/src/util/seo/metaTags.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { getSocialLinks } from 'mwp-app-render/src/util/socialHelper';
+import { generateCanonicalUrl } from './links';
+
+export const FB_APP_ID =
+        process.env.NODE_ENV === 'production' ? '2403839689' : '36212898404';
+
+export const YANDEX_META = {
+	name: 'yandex-verification',
+	value: '68c4b420dba9aa42',
+};
+export const BING_META = {
+	name: 'msvalidate.01',
+	value: '386E470199B7059A6A790A75D5A03B31',
+};
+
+export const TWITTER_APP_ID = '375990038';
+
+export const DEFAULT_IMAGE_URL =
+        'https://secure.meetupstatic.com/s/img/286374644891845767035/logo/meetup-logo-script-1200x630.png';
+
+/**
+ * Strips new lines and tabs from string + truncates provided
+ * @param {String} str String to strip
+ * @param {Number} charLen Number of char to truncate string by
+ * @return {String} new string
+ */
+const stripAndTruncate = (str = '', charLen = 200) =>
+	str
+		.replace(/<(?:.|\n)*?>/gm, '')
+		.replace(/\r?\n|\r\t/g, '')
+		.substr(0, charLen);
+
+/**
+ * Generates array of just topic names based on array of topic objects passed in
+ * @param {Array} topics array of topics
+ * @return {Array} simplified array of topic names
+ */
+export const getTopicNames = topics => topics.map(topic => topic.name);
```

Comment:
not sure this is worth breaking out into a function, but i guess this just part of copy-paste?

Reply:
Yeah, part of the copy / paste. Happy to change though.